v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -348,20 +348,20 @@ protected static java.lang.Class indexTypeToColumnClass(SourceColumnIndexInfo in
   }
 
   /**
-   * Delimit the Column Names as per <a
+   * Delimit the Identifiers as per <a
    * href=https://github.com/ronsavage/SQL/blob/master/sql-99.bnf>sql-99</a>.
    *
    * @param columnName
    * @return
    */
   @VisibleForTesting
-  protected static String delimitColumnName(String columnName) {
+  protected static String delimitIdentifier(String columnName) {
```

Comment:
nit: change internal name in method to identifier as well

Reply:
Thanks. Fixed in current push.